packages/xstate-graph/src/pathFromEvents.ts

```diff
@@ -0,0 +1,87 @@
+import { EventObject } from 'xstate';
+import { isMachine } from 'xstate/lib/utils';
+import { getAdjacencyMap } from './adjacency';
+import {
+  SerializedState,
+  SimpleBehavior,
+  StatePath,
+  Steps,
+  TraversalOptions
+} from './types';
+import {
+  resolveTraversalOptions,
+  createDefaultMachineOptions,
+  createDefaultBehaviorOptions
+} from './graph';
+
+export function getPathFromEvents<
+  TState,
+  TEvent extends EventObject = EventObject
+>(
+  behavior: SimpleBehavior<TState, TEvent>,
+  events: TEvent[],
+  options?: TraversalOptions<TState, TEvent>
+): StatePath<TState, TEvent> {
```

Comment:
We have a method in the `TestModel` that is named exactly like this (in fact, that method calls this function here) **but** it has a different signature. This is kinda confusing - I would expect their APIs to be closer to each other:
https://github.com/statelyai/xstate/blob/eeea324a9fcbc2ac6771fb3b9df3fa97fc073860/packages/xstate-test/src/TestModel.ts#L147-L150

Reply:
This is now unified